### DISCUSSION TITLE
Alternatives for cognition.run

### DISCUSSION THREAD

**Discussion by HannaKdeSouza** [jspsych/jsPsych#2633] [Q&A]
Hello!
I didn't find earlier discussions about this, even though I'm sure they've occurred. Are there other free alternatives to host jsPsych experiments online other than cognition.run? I've used cognition.run very happily in the past and loved its simplicity but as a single researcher without funding, I can't afford to pay for the new premium cognition.run plan. 

The issue I'm facing with the free plan in cognition.run is that my experiments have 270 trials, meaning that I would have to split the task into three in order to be able to run it on cognition.run, thus risking data loss for participants not completing all the parts.

**Comment by jodeleeuw** [comment]
Hi @HannaKdeSouza,

The only completely free option that I know of at the moment is https://mindprobe.eu/. This is a grant-funded [JATOS](https://www.jatos.org/) server, so you'll have to setup your jsPsych experiment to run with JATOS. The JATOS website has [a nice guide for how to do that](https://www.jatos.org/jsPsych-and-JATOS.html).

**Reply by HannaKdeSouza** [reply]
Oh thank you! I had no idea that something like this existed!
